independent-projects/arc/runtime/src/main/java/io/quarkus/arc/AlternativePriority.java

```diff
@@ -4,15 +4,19 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
+import javax.enterprise.inject.Alternative;
 
 /**
  * If a bean is annotated with this annotation, it is considered an enabled alternative with given priority.
  * Effectively, this is a shortcut for {@code Alternative} plus {@code Priority} annotations.
  *
  * This annotation can be used not only on bean classes, but also method and field producers (unlike pure {@code Priority}.
+ * 
+ * @deprecated Use {@link Alternative} and {@link Priority} instead
```

Comment:
Maybe mention FQCN of both `@Priority` annotations? Just to give users a hint. Or at least link both.

Reply:
But these are links...?

Reply:
I meant to link `io.quarkus.arc.Priority` as well as `javax.annotation.Priority`.